source/client/components/CVLanguageManager.ts

```diff
@@ -135,25 +152,37 @@ export default class CVLanguageManager extends Component
 
         return {
             language: ELanguageType[ins.language.getValidatedValue()] as TLanguageType,
+            languages: ins.languages.value.map(l=>ELanguageType[l]) as TLanguageType[],
         };
     }
 
+    /**
+     * Reference a list of languages used in the document
```

Comment:
Not sure I'm understanding the value of this. Can you provide a specific use case?